packages/mui-material/test/integration/Select.test.js

```diff
@@ -62,8 +62,8 @@ describe('<Select> integration', () => {
     it('should be able to change the selected item', () => {
       const { getAllByRole, getByRole, queryByRole } = render(<SelectAndDialog />);
 
-      const trigger = getByRole('button');
-      expect(trigger).toHaveAccessibleName('Ten');
+      const trigger = getByRole('combobox');
+      expect(trigger).toHaveAccessibleName('');
```

Comment:
The accessible name changed here after changing the role from "button" to "combo" box, tho based on the test context, I believe this is correct.

Reply:
I believe this is correct as well, based on [W3 authoring practices](https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-select-only/) and [Base UI's Select](https://mui.com/base-ui/react-select/). The selected value shouldn't be included in the `combobox`'s name.